android/expoview/src/main/java/host/exp/exponent/fcm/ExpoFcmMessagingService.java

```diff
@@ -45,7 +45,7 @@ public void onMessageReceived(RemoteMessage remoteMessage) {
       public void onSuccess(ExperienceDBObject experience) {
         try {
           JSONObject manifest = new JSONObject(experience.manifest);
-          int sdkVersion = ABIVersion.toNumber(manifest.getString("sdkVersion"));
+          int sdkVersion = ABIVersion.toNumber(manifest.getString("sdkVersion")) / 10000;
```

Comment:
I know this is already merged, but for the future I think it's more correct to do something like `if (ABIVersion.toNumber(manifest.getString("sdkVersion")) == ABIVersion.toNumber("38.0.0"))` to abstract away the implementation of `ABIVersion.toNumber`.